teuthology/misc.py

```diff
@@ -550,27 +550,31 @@ def copy_file(from_remote, from_path, to_remote, to_path=None):
     ])
 
 
-def move_file(remote, from_path, to_path, sudo=False):
+def move_file(remote, from_path, to_path, sudo=False, preserve_perms=True):
```

Comment:
I think it is better this way tbh. ```move_file``` already does quite a bit more then its name suggests. Starting to now also set permissions (but only if ```to_path``` doesn't exist) seems to add complexity beyond this functions scope.
Personally I'd expect it to behave like ```mv```, so preserve permissions of the source file. I added the option to keep things compatible with existing code.